config/owasp/suppressions.xml

```diff
@@ -3113,4 +3043,11 @@
         <sha1>e5778e3273f8201bf9e0ff65fb8118f37b84beba</sha1>
         <cve>CVE-2019-17221</cve>
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: flexmark-ext-gitlab-0.34.30.jar
+   ]]></notes>
```

Comment:
can you please try exclude flexmark in build gradle and try if build passes?

Reply:
Done